metadata-ingestion/docs/sources/vertexai/vertexai_pre.md

```diff
@@ -4,7 +4,32 @@ Ingesting metadata from VertexAI requires using the **Vertex AI** module.
 Please refer to the [Vertex AI documentation](https://cloud.google.com/vertex-ai/docs) for basic information on Vertex AI.
 
 #### Credentials to access to GCP
-Please read the section to understand how to set up application default Credentials to GCP [GCP docs](https://cloud.google.com/docs/authentication/provide-credentials-adc#how-to).
+Please read the section to understand how to set up application default Credentials to [GCP docs](https://cloud.google.com/docs/authentication/provide-credentials-adc#how-to).
+
+##### Permissions
+1. Grant the following permissions to the Service Account on every project where you would like to extract metadata from
+
+| Permission                          | Description                                                               | Default GCP Role Which Contains This Permission                                                                 |
+|-------------------------------------|---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| `aiplatform.models.list`            | Allows a user to view and list all ML models in a project                 | [roles/aiplatform.viewer](https://cloud.google.com/vertex-ai/docs/general/access-control#aiplatform.viewer)     |
```

Comment:
should we just say that `roles/aiplatform.viewer` provides all of these permissions at the top? the extra column is a bit weird

alternatively, we should just say that we need the role, and then have a toggle that shows the full table